public/pages/CreateTrigger/components/Action/Action.js

```diff
@@ -96,7 +96,6 @@ const Action = ({
                   </React.Fragment>
                 ),
                 rowHeight: 45,
-                async: true,
```

Comment:
Doubt: Why did we remove the async?

Reply:
@amsiglan, async in this context means filtering will be done manually, look at https://eui.elastic.co/v34.6.0/#/forms/combo-box#async. The prop is set but there is no logic added to support this and hence the combo filter was not working.
The combo gets all the data anyway and the fastest filter here is the local filter which the combo already supports. I removed async in order to use standard combo search.